dev/swift/generate

```diff
@@ -36,6 +36,7 @@ docker run --rm -i -v${PWD}:/code xmtp/protoc-swift \
   message_contents/private_preferences.proto \
   message_contents/signed_payload.proto \
   message_contents/ecies.proto \
+  message_contents/frames.proto \
```

Comment:
I believe Swift no longer uses these scripts and instead generates in `xmtp-ios` using Buf.

Reply:
That's correct, I updated it for parity, but can remove it

Reply:
We should just axe the whole thing, but can do that in another PR